frontend/javascripts/admin/organization/upgrade_plan_modal.tsx

```diff
@@ -0,0 +1,196 @@
+  const extendedDate = moment(organization.paidUntil).add(1, "year");
+
+  Modal.confirm({
+    title: "Extend Current Plan",
+    okText: "Request Extension",
+    onOk: sendExtendPricingPlanEmail,
+    icon: <FieldTimeOutlined style={{ color: "var(--ant-primary-color)" }} />,
+    width: 1000,
+    content: (
+      <div
+        style={{
+          background:
+            "right -150px / 50% no-repeat url(/assets/images/pricing/background_neurons.png)",
+        }}
+      >
+        <p style={{ marginRight: "30%" }}>
+          Extend your plan now for uninterrupted access to webKnossos. Expired plans will be
+          downgraded to the Free plan and you might lose access to some webKnossos features and see
+          restrictions on the number of permitted user accounts and your included storage space
+          quota.
+        </p>
+        <p>
+          Your current plan is paid until:{" "}
+          {formatDateInLocalTimeZone(organization.paidUntil, "YYYY-MM-DD")}
+        </p>
+        <p>Buy extension until: {extendedDate.format("YYYY-MM-DD")}</p>
+        <Divider style={{ marginTop: 40 }} />
+        <p style={{ color: "#aaa", fontSize: 12 }}>
+          Requesting an extension will send an email to the webKnossos sales team to extend your
+          current plan. We typically respond within one business day. webKnossos plans are billed
+          annually. See our <a href="https://webknossos.org/faq">FAQ</a> for more information.
+        </p>
+      </div>
+    ),
+  });
+}
+
+function upgradeUserQuota() {
+  const userInputRef = useRef<HTMLInputElement | null>(null);
+
+  function handleUserUpgrade() {
+    const requestedUsers = userInputRef.current?.value;
+    sendUpgradePricingPlanUserEmail(requestedUsers);
+  }
+
+  Modal.confirm({
+    title: "Upgrade User Quota",
+    okText: "Request More Users",
+    onOk: handleUserUpgrade,
+    icon: <UserAddOutlined style={{ color: "var(--ant-primary-color)" }} />,
+    width: 1000,
+    content: (
+      <div
+        style={{
+          background:
+            "right -50px / 35% no-repeat url(/assets/images/pricing/background_users.png)",
+        }}
+      >
+        <p style={{ marginRight: "30%" }}>
+          You can increase the number of users allowed to join your organization by either buying
+          single user upgrades or by upgrading your webKnossos plan to “Power” for unlimited users.
+        </p>
+        <div>Add additional user accounts:</div>
+        <div>
+          <InputNumber min={1} defaultValue={1} ref={userInputRef} />
+        </div>
+
+        <Divider style={{ marginTop: 40 }} />
+        <p style={{ color: "#aaa", fontSize: 12 }}>
+          Requesting an upgrade to your organization&apos;s user quota will send an email to the
+          webKnossos sales team. We typically respond within one business day. See our{" "}
+          <a href="https://webknossos.org/faq">FAQ</a> for more information.
```

Comment:
DRY this with the other components?

Reply:
The text is slightly different for each of the modals depending on your action. Not sure how you would DRY that?

Reply:
Maybe just make it the same for all modals (except extend):
```
Requesting an upgrade to your organization will send an email to the webKnossos sales team. We typically respond within one business day with next steps. Billing occurs annually. See our <a href="https://webknossos.org/faq">FAQ</a> for more information.```